src/main/scala/org/broadinstitute/dsde/workbench/sam/model/api/SamJsonSupport.scala

```diff
@@ -101,7 +101,7 @@ object SamJsonSupport {
 
   implicit val UserPolicyResponseFormat = jsonFormat5(UserPolicyResponse.apply)
 
-  implicit val UserUpdateRequestFormat = jsonFormat3(AdminUpdateUserRequest.apply)
```

Comment:
I'm not too familiar with Scala, but Is there a reason why this is using `jsonFormat4` now? I see some other versions being used in this file so I'm wondering if we should standardize this in a follow-on ticket?

Reply:
yeah good question, it is because this is the function that automatically parses a json blob to the AdminUpdateUserRequest object, I added a new field to AdminUpdateUserRequest so now it is 4 arguments.